parlai/core/params.py

```diff
@@ -809,14 +809,26 @@ def add_task_args(self, task):
         Add arguments specific to the specified task.
         """
         for t in ids_to_tasks(task).split(','):
-            agent = get_task_module(t)
+            agent = load_teacher_module(t)
             try:
                 if hasattr(agent, 'add_cmdline_args'):
                     agent.add_cmdline_args(self)
             except argparse.ArgumentError:
                 # already added
                 pass
 
+    def add_world_args(self, task, interactive_task):
+        """
+        Add arguments specific to the world.
+        """
+        world_class = load_world_module(task, interactive_task)
+        if world_class is not None and hasattr(world_class, 'add_cmdline_args'):
+            try:
+                world_class.add_cmdline_args(self)
+            except argparse.ArgumentError:
+                # already added
```

Comment:
does this mean that the world is trying to add arguments that have already been added to the argparser?

Reply:
yes. i'm not sure when this happens, but it was in the previous code